tests/middleware/test_token_cost.py

```
@@ -0,0 +1,183 @@
+            "messages": [ai_message],
+            "todos": None,
+            "files": {},
+            "current_input_tokens": None,
+            "current_output_tokens": None,
+            "total_cost": None,
+        }
+
+        runtime = Mock()
+        runtime.context = AgentContext(
+            approval_mode=ApprovalMode.SEMI_ACTIVE,
+            working_dir=Path("/tmp"),
```

Ruff (0.14.3)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)